content/docs/guides/continuous-delivery/spinnaker.md

```diff
@@ -101,98 +126,126 @@ To create this resource in your Spinnaker cluster, run:
-  * As previously mentioned in this guide, you can use [pipeline expressions](https://www.spinnaker.io/guides/user/pipeline/expressions/) for your stage inputs.
+  * As previously mentioned in this guide, you can use [pipeline expressions](https://www.spinnaker.io/guides/user/pipeline/expressions/)
+    for your stage inputs.
 * Note that we are setting the value of the **Command** input to `preview`.
 
-![Pulumi Stage](/images/docs/guides/continuous-delivery/spinnaker/pipeline-stage.png)
-
-Add a [Manual Judgement](https://www.spinnaker.io/guides/tutorials/codelabs/safe-deployments/#adding-a-manual-judgment-to-deployment-pipelines) stage to the pipeline and then let's add the Pulumi stage again, but this time we'll set the **Command** input to `up` instead of `preview`.
-In the end, your pipeline should look like this:
+Add a [Manual Judgement](https://www.spinnaker.io/guides/tutorials/codelabs/safe-deployments/#adding-a-manual-judgment-to-deployment-pipelines)
+stage to the pipeline and then let's add the Pulumi stage again, but this time we'll set the **Command** input to `up`
+instead of `preview`. In the end, your pipeline should look like this:
 
 ![Full Pipeline](/images/docs/guides/continuous-delivery/spinnaker/full-pipeline.png)
 
 Go back to the main Pipelines view and click on the **Start Manual Execution** link next to your pipeline.
-Check the logs from Pulumi as the Pulumi stage runs. The logs update in near real-time.
+Once the job starts execution, you should see a **Console Output** link. Clicking it will open a modal that will show
+you the current output.
 
-![Pulumi Stage](/images/docs/guides/continuous-delivery/spinnaker/pipeline-logs.png)
+**Note**: If you are running Spinnaker on your machine, it can take a while before the job starts to execute and the logs
+are visible. If you see an error about `ContainerCreating` it means the pod running the job isn't ready yet.
 
-Once the pipeline is complete, assuming you have not modified the example app, the `nginx` container would have been deployed to the `default` namespace.
-Get the pod IP by running a `kubectl` command and `curl`-ing the IP to see the response:
+Once the pipeline is complete, assuming you have not modified the example app, the `nginx` container would have been
+deployed to the `default` namespace. Get the pod IP by running a `kubectl` command and `curl`-ing the IP to see the response:
 
 ```
 kubectl get pods -o wide
 
-curl http://POD_IP
+curl http://<POD_IP>
 ```
 
+## FAQs
+
+**Can I get colored log output from Pulumi?**
+
+Yes! Simply pass the Pulumi CLI arg `--color always`. Now when you open the `Console Output` modal, you should see
+colors in the log output. Note that the colored logs are only visible starting with v0.2.0 of the plugin.
+
+**How do I use the plugin with an enterprise VCS repo such as GitHub Enterprise?**
+
+The plugin allows you to specify custom container images. One option is to build the container image with the SSH
```

Comment:
do we really want to suggest this?  (maybe just suggest the token approach?)

Reply:
I am a little conflicted on this myself. But if the image is private, then it should be fine. But of course, it's up to the Spinnaker admin to decide whether or not they want to consider this approach.

@leezen what do you think about mentioning this part?